test/Common/Helpers/AssertHelper.cs

```diff
@@ -45,6 +46,18 @@ public static void AssertPropertyEqual(object expected, object actual, params st
                 static string JsonDocString(PropertyInfo info, object obj) => JsonSerializer.Serialize(info.GetValue(obj));
                 Assert.Equal(JsonDocString(expectedPropInfo, expected), JsonDocString(actualPropInfo, actual));
             }
+            else if (typeof(IEnumerable).IsAssignableFrom(expectedPropInfo.PropertyType) && typeof(IEnumerable).IsAssignableFrom(actualPropInfo.PropertyType))
```

Comment:
I wrote an overload for `AssertProppertyEqual` for `IEnumerable`s here: https://github.com/bitwarden/server/blob/4be9a8999eb379231d53bbcf65d80fced4e960a2/test/Common/Helpers/AssertHelper.cs#L80-L80

Can we reuse that instead?

Reply:
I didn't notice that that exists. Updated to use it.